src/modules/pivotTable/useParentSize.js

```diff
@@ -20,13 +21,18 @@ export const useParentSize = (
                 height: el.clientHeight,
             })
         }
+
         onResize(el)
 
+        if (renderId) {
```

Comment:
Do we need to check `renderId` at all here?  It should also work to do this on the first `effect` call, so long as `renderId` is kept in the dependency array

Reply:
`renderId` is `null` in the first render when the interpretations button has not been clicked yet; this was to avoid resetting the width in that case.

Reply:
If it is `null` on first render is it `0` or `1` on first update?  If `0` this won't trigger an update until the third render...

Reply:
Starts with 1.